worker/src/test/java/gov/cms/ab2d/worker/adapter/bluebutton/ContractAdapterTest.java

```diff
@@ -0,0 +1,59 @@
+package gov.cms.ab2d.worker.adapter.bluebutton;
+
+import gov.cms.ab2d.bfd.client.BFDClient;
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Patient;
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+class ContractAdapterTest {
```

Comment:
We need to "beef up" this test significantly. Mock various responses from BFD and assert the results are as expected. Examples:

- A Patient is returned for month 1 & 2, verify its date ranges set correctly
- A Patient is returned for month 1, 3, but not 2 -- verify the date ranges have a gap as expected
- A Patient is returned multiple times for a single month -- ensure we handle this by filtering out dupes. 
- Simulate paging: first mock Bundle has a paging link that leads to the 2nd bundle: verify that Patients from both bundles are collected.

Reply:
added additional tests

Reply:
done